docs/ru-RU/README.md

```diff
@@ -3,7 +3,7 @@
 [travis-link]: https://travis-ci.org/fisherman/fisherman
 [travis-badge]: https://img.shields.io/travis/fisherman/fisherman.svg
 
-[fish shell]: https://github.com/fish-shell/fish-shell
+[fish-shell]: https://github.com/fish-shell/fish-shell
```

Comment:
This breaks the link in the first heading. Could you remove this dash? Or add it in the heading link. I don't know what is the canonical form to refer to fish..